wowchemy/assets/scss/wowchemy/_breadcrumb.scss

```diff
@@ -0,0 +1,15 @@
+.breadcrumb {
+  font-size: 14px;
```

Comment:
Please use REM units

Reply:
Done

Reply:
Thanks. The changes aren't showing up in the PR...

Reply:
Sorry, forgot to push yesterday.